app/models/manageiq/providers/hawkular/middleware_manager/middleware_domain.rb

```diff
@@ -0,0 +1,4 @@
+module ManageIQ::Providers
```

Comment:
aren't the new models extending ActiveRecord or I misunderstood something? (like, for example, middleware_deployment, middleware_server, etc.)